packages/core/src/__tests__/ceramic-pinning.test.ts

```diff
@@ -271,17 +294,56 @@ describe('Ceramic stream pinning', () => {
     await ceramic.close()
   })
 
-  it('Stream can be pinned and unpinned on load', async () => {
+  it('Stream cannot be pinned and unpinned on load', async () => {
     const ceramic = await createCeramic(ipfs1, tmpFolder.path)
     const stream = await TileDocument.create(ceramic, { foo: 'bar' }, null, {
       anchor: false,
       publish: false,
       pin: false,
     })
+
     await expect(TestUtils.isPinned(ceramic, stream.id)).resolves.toBeFalsy()
-    await TileDocument.load(ceramic, stream.id, { sync: SyncOptions.NEVER_SYNC, pin: true })
+    // pin:true flag will be ignored
+    TileDocument.load(ceramic, stream.id, { sync: SyncOptions.NEVER_SYNC, pin: true })
+    await expect(TestUtils.isPinned(ceramic, stream.id)).resolves.toBeFalsy()
+    await ceramic.admin.pin.add(stream.id)
     await expect(TestUtils.isPinned(ceramic, stream.id)).resolves.toBeTruthy()
-    await TileDocument.load(ceramic, stream.id, { sync: SyncOptions.NEVER_SYNC, pin: false })
+    // pin:false flag will be ignored
+    TileDocument.load(ceramic, stream.id, { sync: SyncOptions.NEVER_SYNC, pin: false })
+    await expect(TestUtils.isPinned(ceramic, stream.id)).resolves.toBeTruthy()
+
+    await ceramic.close()
+  })
+
+  it('Existing stream cannot be pinned and unpinned on deterministic load', async () => {
```

Comment:
Why not pinned on load? 
Scenario: 
1. Someone creates a deterministic stream for user A without pinning
2. User A now creates their stream and tries to pin it
3. Operation 2 fails, this seems problematic

Reply:
@stbrody this needs to be fixed imo

Reply:
This should work I believe, so long as there's still only a single commit on the stream, you can pin with the `deterministic` api.  Once an update has been added, then you can no longer pin it

Reply:
Ok, then I'm fine with this PR 👍

Reply:
Added a test case to demonstrate